### DISCUSSION TITLE
error when instaling keras and tensorflow in R using apple M1 chipset

### DISCUSSION THREAD

**Discussion by tatapinandhika** [teamalgoritma/community#172] [Q&A] [answered]
Saya telah melakukan berbagai cara. tapi masih error, ini cara terakhir yang disarankan oleh mentor algoritma, tetapi tetap error.

<img width="1440" alt="Screen Shot 2022-07-03 at 21 04 39" src="https://user-images.githubusercontent.com/101564223/177041579-0199cdb6-3d5f-478a-be75-a5173f7c62c0.png">

<img width="1440" alt="Screen Shot 2022-07-03 at 21 15 47" src="https://user-images.githubusercontent.com/101564223/177041599-8be22b73-b9ca-4b4c-b17f-4d6b592fbc5f.png">

**Comment by ranydwi** [comment, marked as answer]
Dear @tatapinandhika 

Permasalahan yang Bapak alami terjadi karena memang penggunaan Tensorflow dan Keras pada chipset M1 masih belum disupport. Akan tetapi berdasarkan kasus Bapak, beberapa hal yang dapat bapak coba yaitu:
1. Sampai official support tersedia, Apple menyediakan build khusus Tensorflow khusus untuk M1 Mac. Anda dapat menginstalnya dengan mengikuti petunjuk di sini: https://developer.apple.com/metal/tensorflow-plugin/
sumber: https://stackoverflow.com/questions/69898402/i-am-not-able-to-use-functions-in-keras-on-m1-mac-in-r-and-it-is-asking-me-to-in
2. Bapak dapat menggunakan Kaggle Kernel apabila ingin menggunakan Tensorflow dan Keras untuk pengerjaan LBB/kebutuhan lain yang sesuai dengan tujuan Bapak.
<img width="960" alt="image" src="https://user-images.githubusercontent.com/68055107/177257433-dc1e00c5-a20f-47ea-895d-9617a96fecea.png">

Semoga membantu,
Regards.
